pkg/executor/common.go

```diff
@@ -557,7 +557,7 @@ func GetPodErrorMessage(ctx context.Context, client kubernetes.Interface, pod *c
 
 	for _, containerStatus := range pod.Status.ContainerStatuses {
 		if containerStatus.State.Terminated != nil &&
-			(containerStatus.State.Terminated.ExitCode > 1 || containerStatus.State.Terminated.ExitCode < -1) &&
+			(containerStatus.State.Terminated.ExitCode >= 1 || containerStatus.State.Terminated.ExitCode < -1) &&
```

Comment:
then also for init container above. but the idea was for 1 and -1 to show test error message from the log, because it's just failed test

Reply:
There is empty message in real time long case of `1`

Reply:
ok, it's fine for me, if @tkonieczny has no objections

Reply:
ok added also in init container